features/FEATURE_CLIENT/mbed-client-classic/source/m2mconnectionhandlerpimpl.cpp

```diff
@@ -310,9 +312,7 @@ int8_t M2MConnectionHandlerPimpl::connection_tasklet_handler()
 
 void M2MConnectionHandlerPimpl::socket_event()
```

Comment:
is this called from irq context? then it make sense to minimize code here, like remove trace and allocation related calls,,, but how faster this memory_pool really are? any numbers?